report/MVP/MVP.tex

```diff
@@ -13,18 +13,22 @@ \section{What is our MVP?}
 
 The purpose of this MVP is to give the PO an idea of how to formalize their requirements, in a mobile app, meaning that they can use it anytime they want, and anywhere, as long as they have a connection to the internet.
 Furthermore, it will also give the developers an idea of how the workflow will be like, and how they can get more or less instant feedback, regarding if they have solved the task at hand, or not. 
-The app will be basic, however, it will showcase how to use it, and what the progress is.
+The app will be basic, however, it will showcase how to use it, and what the progress is. 
+
+This MVP will only focus on the UI elements, meaning that we will not make any implementations in regards to the class invariant, which is proposed in~\autoref{sec:DbC}.
+The reason why, is that we see more value gathered from the workflow of the application, where the PO can create and edit screens.
+Furthermore, it will not give us much value in terms of the learning experience, as this is somewhat trivial to do, whereas the critical features are more complex and give us more learning experience, in our opinion.
```

Comment:
Her har vi igen som Ivan har sagt, siger vi at noget er "somewhat trivial" men forklare ikke hvorfor

Reply:
True. Kunne man gøre noget ala dette?
```suggestion
Furthermore, it will not give us much value in terms of the learning experience, as this is somewhat trivial to do, whereas the critical features are more complex and give us more learning experience, in our opinion.
The reason why we believe that this is trivial, is that checking if a given element is still on the screen, is straightforward to do in Flutter.
```